curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b6217f2fa8a06818fd8aa.md

```diff
@@ -0,0 +1,147 @@
+---
+id: 646b6217f2fa8a06818fd8aa
+title: Step 25
+challengeType: 0
+dashedName: step-25
+---
+
+# --description--
+
+You will target the class named `.select-icon` in the CSS file and add the following properties:
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed